MonoZelda/UI/HUDMapWidget.cs

```diff
@@ -42,6 +43,12 @@ public HUDMapWidget(Screen screen, Point position) : base(screen, position)
         compassMarker = new(SpriteType.HUD, SpriteLayer.HUD + 1, WidgetLocation + compassMarkerOffset);
         compassMarker.SetSprite("map_marker_red_blinking");
         compassMarker.Enabled = compassMarkerEnabled;
+        compassItem = new(SpriteType.HUD, SpriteLayer.HUD + 1, WidgetLocation + mapItemSpriteOffset);
+        compassItem.SetSprite("compass");
+        compassItem.Enabled = compassMarkerEnabled;
+        mapItem = new(SpriteType.HUD, SpriteLayer.HUD + 1, WidgetLocation + compassItemSpriteOffset);
+        mapItem.SetSprite("map");
+        mapItem.Enabled = mapEnabled;
```

Comment:
The compass and map sprites should go in `InventoryMapWidget`, not `HUDMapWidget`. The HUD is the part of the inventory you see during normal gameplay, and those sprites are only displayed in the full screen inventory.